docs/user-guide/masking.ipynb

```diff
@@ -55,15 +55,33 @@
     "print(mask | ~mask)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Comparison operators such as `==` or `<` are a common method of defining masks:"
+   ]
```

Comment:
Should we link to the full supported list of comparison operations?

Reply:
Done.